templates/compass.html

```diff
@@ -45,6 +49,22 @@ <h4>
                                     <li>Select 'Upload from local' and upload the generated layer file</li>
                                 </ol>
                             </div>
+                            <br>
+                            <div style="background-color: var(--primary-background);padding: 15px;border-radius: 25px">
+                                <h4>
+                                Create an Adversary to use in an operation. Follow the steps below to create your own
+                                    Adversary:
+                                </h4>
+                                <ol>
+                                    <li>Select Techniques: Right Click and 'add to selection' </li>
+                                    <li>Once all techniques are selected, change the score (bar chart under 'technique
+                                        controls' to a positive number</li>
```

Comment:
This is an overwhelming amount of text when you load the page. I'd leverage the ?/duk pop ups to hide some of it and show only what you need to. or reword/restructure what you're trying to get across.

Reply:
done see image below